admin/actions.php

```diff
@@ -72,60 +72,34 @@ function hmbkp_request_do_backup() {
 		check_admin_referer( 'hmbkp_run_schedule', 'hmbkp_run_schedule_nonce' );
 	}
 
-	// Fixes an issue on servers which only allow a single session per client
-	session_write_close();
```

Comment:
maybe, Backdrop calls `wp_remote_post` on the admin-ajax file. 
I added it back